sajibportfolio.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="/sajibportfolio.css" />
+    <title>Finish Mobile Version</title>
+  </head>
+
+  <body>
+    <section id="works">
+      <section id="my_work">
+        <div class="snapshot_portfolio"></div>
+          <div id="ptext">
+              <h2 class="ptitle">Sajib</h2>
+              <ul class="tags">
+                <li>
+                  <p>CANOPY</p>
+                </li>
+                <li>
+                  <p>Back End Dev</p>
+                </li>
+                <li>
+                  <p>2015</p>
+                </li>
+              </ul>
+          
+             
+              <p class="pdesc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            
+          
+            <ul class="skills">
+              <li class="skill1">
+                <p class="title1">HTML</p>
+              </li>
+              <li class="skill2">
+                <p class="title2">CSS</p>
+              </li>
+              <li class="skill3">
+                <p class="title3">JAVASCRIPT</p>
+              </li>
+            </ul>
```

Comment:
Your overall design is still far from the Figma template provided. Kindly fix all the issues raised by the previous and try to follow the Figma design almost exactly as given. See your design now. Please look at the template again and follow.
![image](https://user-images.githubusercontent.com/84156714/205410796-5c803a7f-23a8-484e-b0d8-30d1c9f1979a.png)

Reply:
I have tried my best now to make almost similar to the figma design.